SimplenoteUITests/NoteEditor.swift

```diff
@@ -177,6 +185,12 @@ class NoteEditor {
         return matches
     }
 
+    class func openHistory() {
+        app.navigationBars[UID.NavBar.allNotes].buttons[UID.Button.noteEditorMenu].tap()
```

Comment:
I know this is the same in older code, so it's not technically incorrect. This is just something that we have to keep in mind when writing new code (and once we have time to refactor the old code). 

The navigation bar title depends on the current selection in note list. For example, if we open `All Notes`, the navbar title will be `All notes`. If we create a tag for a note, and open all notes tagged with a certain tag, the navbar title will be this tag, but not `All Notes`.

Because of this, hardcoding the navbar title in the button locator limits us to the usage of this method to only `All Notes`. This can be avoided just by removing the navbar part:

```suggestion
        app.buttons[UID.Button.noteEditorMenu].tap()
```